R/exercise_0.Rmd

```diff
@@ -45,15 +46,41 @@ In this case, we want to modify the general pattern with:
 - `[Resource] = Patient`
 - [`_id` parameter](https://www.hl7.org/fhir/search.html#id) = `smart-1032702`
 
+```{r}
+fhir_server <- "https://api.logicahealth.org/researchonfhir/open/"
+cs <- fhir_capability_statement(fhir_server)
+##?cs$Resources$typestr_c("FHIR Resources Supported: ", cs$Resources$type %>% length, "\n") %>% cat
+```
+
 ```{r}
 fhir_server <- "https://api.logicahealth.org/researchonfhir/open/"
 request <- fhir_url(url = fhir_server, resource = "Patient", parameters = list("_id"="smart-1032702"))
-patient_bundle <- fhir_search(request = request)
-patient_bundle
+patient_bundle1 <- fhir_search(request = request)
+patient_bundle1
+request2 <- fhir_url(url = fhir_server, resource = "Patient", parameters = list("_id"="BILIBABY"))
+patient_bundle2 <- fhir_search(request = request2)
+patient_bundle2
 ```
 
+
```

Comment:
Should we add some explanation about the differences between these FHIR searches (above and below)?